src/parser/DefaultParser.js

```diff
@@ -29,6 +29,16 @@ class DefaultParser {
     this.pluginData.parseErrors = [];
   }
 
+  /**
+   * Get the list of models from all parsable files.
```

Comment:
Get the list of model paths from all parsable files.

Reply:
done